share/sysbuild/cmake/modules/sysbuild_extensions.cmake

```diff
@@ -353,6 +363,93 @@ function(ExternalZephyrProject_Cmake)
   import_kconfig(CONFIG_ ${BINARY_DIR}/zephyr/.config TARGET ${ZCMAKE_APPLICATION})
 endfunction()
 
+# Usage:
+#   ExternalZephyrProject_Group(<name>
+#                               [INCLUDE <images>]
+#                               [EXCLUDE <images>]
+#                               [INCLUDE_FROM <groups>]
+#                               [EXCLUDE_FROM <groups>]
+#                               [IMMUTABLE]
+#   )
+#
+# This function defines groups of external Zephyr projects. Images added using
+# `ExternalZephyrProject_Add()` may inherit common properties of the group(s)
+# they are assigned to.
```

Comment:
> may inherit common properties of the group(s) they are assigned to.

this sound a bit fluffy.
Do they inherit properties, and if so, which properties a inherited ?

This starts to sound a bit like CMake's [Transitive Usage Requirements](https://cmake.org/cmake/help/latest/manual/cmake-buildsystem.7.html#target-usage-requirements), and if that is the case, then we should consider if those key words would be better to use in this context.